test/ui/test/html-report.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { resolve } from 'node:path'
+import { beforeAll, expect, it } from 'vitest'
+import { execaCommandSync } from 'execa'
+import { browserErrors, page, ports, startServerCommand } from '../setup'
+
+const root = resolve(__dirname, '../fixtures')
+const port = ports.report
+
+beforeAll(async () => {
+  execaCommandSync('npx vitest run --reporter=html --outputFile=html/index.html', {
+    cwd: root,
+    env: {
+      ...process.env,
+      CI: 'true',
+      NO_COLOR: 'true',
+    },
+  })
+
+  const exit = await startServerCommand(
+    root,
+    `npx vite preview --outDir html --strict-port --base /__vitest__/ --port ${port}`,
+    `http://localhost:${port}/__vitest__/`,
+  )
+
+  return exit
+})
+
+it('should load report', async () => {
+  expect((await (await page.$('#app'))?.innerHTML() || '').length).not.toBe(0)
```

Comment:
can we add a more elaborate check here? maybe `toMatch` and check for some HTML there